scripts/tests/auth/test-self-hosted.sh

```diff
@@ -23,6 +23,9 @@ fi
 earthly=${earthly:=earthly}
 earthly=$(realpath "$earthly")
 echo "running tests with $earthly"
+echo "using frontend $frontend"
```

Comment:
nit: can we move this after the `frontend="${frontend:-$(which docker || which podman)}"`, so we can see the final value (after `which`)